backend/src/main/java/com/rikishi/rikishi/api/HealthcheckController.java

```diff
@@ -0,0 +1,12 @@
+package com.rikishi.rikishi.api;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthcheckController {
+    @GetMapping("/health")
+    public Healthcheck healthcheck() {
```

Comment:
I think it will be much readable to name functions in a self-explanatory way, i.e. for get mappings: `getHealth`, for post mappings `setHealth` etc

Reply:
The name of this function doesn't really matter, as it's unlikely to be called anywhere else, but yeah I agree with your sentiment